packages/react-magma-docs/src/pages/api/select.mdx

```diff
@@ -795,6 +796,70 @@ interface CustomSelectItem {
 }
 ```
 
+## Custom Item Render
+
```

Comment:
I feel like this could use a paragraph describing what this does (and how it's different from the other custom examples)